go/kbnm/hostmanifest/whitelist_nix.go

```diff
@@ -0,0 +1,17 @@
+// +build !darwin
```

Comment:
Windows is not an issue. It can build for Windows, it's just not in the Windows codepath. I can !windows if you think that would be better?